src/MetaMaskInpageProvider.rpc.test.ts

```diff
@@ -636,4 +636,42 @@ describe('MetaMaskInpageProvider: RPC', () => {
       );
     });
   });
+  describe('provider events', () => {
+    // eslint-disable-next-line jest/no-test-callback
+    it('calls chainChanged when it chainId changes ', (done) => {
+      const mockStream = new MockDuplexStream();
+      const p = new MetaMaskInpageProvider(mockStream);
```

Comment:
nit: Single-character variable names serve the devil's workshop.

```suggestion
      const provider = new MetaMaskInpageProvider(mockStream);
```

Reply:
`provider` is a variable name in the above scope

Reply:
changed to `inpageProvider`